fah_xchem/analysis/website/templates/transformations/index.html

```diff
@@ -12,12 +12,12 @@ <h3>Transformations</h3>
 </div>
 <table class="table table-striped table-hover">
   <tr>
-    <th>RUN</th>
-    <th colspan=4>Initial microstate</th>
-    <th colspan=4>Final microstate</th>
-    <th>ΔΔG / kcal M<sup>-1</sup></th>
-    <th>Work distributions</th>
-    <th>Convergence</th>
+    <th>RUN <i type="button" class="fa fa-info-circle" data-html="true" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="bottom" data-title="<b>RUN</b>" data-content="The unique run analysing the transformation between the initial and final microstate. <br /><br /> Each RUN consists of a number of iterations (GENs)."></i></th>
+    <th colspan=4>Initial microstate <i type="button" class="fa fa-info-circle" data-html="true" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="bottom" data-title="<b>Initial microstate</b>" data-content="The unique identifier of the initial microstate."></i></th>
+    <th colspan=4>Final microstate <i type="button" class="fa fa-info-circle" data-html="true" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="bottom" data-title="<b>Final microstate</b>" data-content="The unique identifier of the final microstate."></i></th>
+    <th>ΔΔG / kcal M<sup>-1</sup> <i type="button" class="fa fa-info-circle" data-html="true" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="bottom" data-title="<b>Relative binding free energy</b>" data-content="The calculated relative binding free energy of the transformation between the inital and final microstate."></i></th>
```

Comment:
"The computed relative free energy for transforming the initial microstate into the final microstate. Negative ΔΔG values indicate the final microstate binds more favorably than the initial microstate, while positive ΔΔG values indicate the final microstate binds less favorably. Note that the ΔΔG values reported here will not be identical to differences in ΔG reported for microstates because DiffNet is used to account for thermodynamic self-consistency in reported microstate absolute ΔG values."